### ISSUE TITLE
Add "Stay local" feature in the admin control  panel

### ISSUE THREAD

**Issue by tiotrom** [friendica/friendica#9960]
We are going to release our Friendica node soon, trying to get more people to use this awesome tool you guys developed. One thing that I saw with everyone I showed Friendica too, is that they naturally click people's names to go to their "page" and interact with them. But by default, in Friendica, when one does that, it is sent to the user's home-town. Could be a different Friendica node with a  completely different look, a mastodon account, peertube and what not. And everyone is confused from what I observed. 

So, although the "stay local" feature is an opt-in thing, I suggest to allow admins to enable it if they wish to.
Thanks!

**Comment by MrPetovan** [comment]
Follow-up to #9331

**Comment by annando** [comment]
The way it is done right now does mean that this new setting would disable the user's choice. Otherwise we had to move it to the advanced settings.

**Comment by tiotrom** [comment]
Am not sure I understand Michael :D 

There are many Admin settings that interfere with the user's choice, but if the user can change back the option in their settings, should be fine, right?

**Comment by MrPetovan** [comment]
The way you're describing it would be an "Additional Feature". The node admin can set a node-wide value, and can allow or prevent users from overriding it on their account.

**Comment by tiotrom** [comment]
> The way you're describing it would be an "Additional Feature". The node admin can set a node-wide value, and can allow or prevent users from overriding it on their account.

Yes exactly. And this is more difficult to implement you are saying?

**Comment by MrPetovan** [comment]
Slightly, but it's mostly a decision we have to make. Adding an Additional Feature makes it harder to find it both for admins and users alike because they have their dedicated page away from the regular settings, so there's a usability penalty for them at the moment.

**Comment by annando** [comment]
@MrPetovan yeah, exactly. This is one of the reasons why I haven't done it by now. Perhaps we should rebuilt the settings so that all settings are grouped for their purpose and "additional settings" and addon settings would be grouped into this automagically as well.

**Comment by tiotrom** [comment]
I see. I find it confusing a bit with the settings and such. I think what can help a lot is a quick search filter that can display only the tables that contain the filter word. I know is easier said than done, but just my 2c. In any case, I think the "stay local" is very useful for most users and I'd really like to be able to enable it by default for our node :). But you people are too friendly and awesome, so whenever you have time and if you think is worth it ;). I wish I had the time and skills to help out directly. Maybe in the future.

**Comment by annando** [comment]
I would like to move it to a milestone where we reworked the whole settings to make them more user friendly.

**Comment by MrPetovan** [comment]
It isn't in a milestone yet, so we're good on that front.

**Comment by hoergen** [comment]
The "stay local" feature should be reworked from ground, so there is a clear and intuitive UX.

There are several issues with "stay local" and it is totally confusing. Even in your contacts clicking on a link for an external profile `https://friendica.node/external/profile` sends you to the local profile. That should not be the case.  Instead, you should jump to the external site (WYSIWYG)

![grafik](https://user-images.githubusercontent.com/24474762/114297883-8409f280-9ab3-11eb-918f-5ea02ca3e0cf.png)



Issues I found
- stay local not working on mention links https://github.com/friendica/friendica/issues/9559
- Locally redirect contacts https://github.com/friendica/friendica/issues/9524
- Setting "stay local" doesn't work on remote servers https://github.com/friendica/friendica/issues/9435

**Comment by xundeenergie** [comment]
Yesterday I started using friendica after 2 years again... and now I'm using it... and a just would open a feature-request for this. It's annoying, that clicking on a profile leads me to the original profile-page... 

"Stay home" should absolutely be the default, and "show original profile home" should be the opt-in.

**Comment by MrPetovan** [comment]
Thank you for your input. like @annando said, changing the default behavior node-wide would be a complete reversal of the original Friendica philosophy and as such we won't do it lightly.

**Comment by annando** [comment]
A possibility would be to have a system setting for this setting for new users. But I wouldn't want to change thos behaviour for existing users.

**Comment by tio-trom** [comment]
I'd say adding it as an option for the instance admins, would be the best for now. I personally find it extremely useful and we would love to enable it by default for our node ;).

**Comment by marcusxss** [comment]
This is about wanting to be able to enable "Stay local", and "Stay local" is now default, so maybe this can be closed?
#15057

**Comment by tio-trom** [comment]
Yup. Perfect. I do not have access to the previous account anymore so if you guys can please close this. Thanks

**Comment by marcusxss** [comment]
Great!

@nupplaphil ? 😃

**Comment by nupplaphil** [comment]
Closed with #15057
